File: solutions/uri/1012/1012.c
#include <stdio.h>

#define pi 3.14159

int16_t main() {
    float a, b, c;

    while (scanf("%f %f %f", &a, &b, &c) != EOF) {
        printf("TRIANGULO: %.3f\n", a * c / 2);
        printf("CIRCULO: %.3f\n", pi * (c * c));
        printf("TRAPEZIO: %.3f\n", c * ((a + b) / 2));
        printf("QUADRADO: %.3f\n", b * b);
        printf("RETANGULO: %.3f\n", a * b);
    }
}

File: solutions/uri/1012/1012.py
import sys

PI = 3.14159

for line in sys.stdin:
    a, b, c = map(float, line.split())

    print(f'TRIANGULO: {a * c / 2:.3f}')
    print(f'CIRCULO: {PI * (c * c):.3f}')
    print(f'TRAPEZIO: {c * ((a + b) / 2):.3f}')
    print(f'QUADRADO: {b * b:.3f}')
    print(f'RETANGULO: {a * b:.3f}')
